src/win32/posix.h

```diff
@@ -4,14 +4,15 @@
  * This file is part of libgit2, distributed under the GNU GPL v2 with
  * a Linking Exception. For full terms see the included COPYING file.
  */
-#ifndef INCLUDE_posix__w32_h__
-#define INCLUDE_posix__w32_h__
+#ifndef INCLUDE_posix_w32_h__
+#define INCLUDE_posix_w32_h__
```

Comment:
This seems unnecessary.  Can we drop this?  It looks like we have some issues around consistency with these guard names, but let's fix that in a separate PR instead of piecewise.